config/crd/scaling.core.adobe.com_kratos.yaml

```diff
@@ -1,10 +1,10 @@
 
 ---
-apiVersion: apiextensions.k8s.io/v1beta1
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
small nitpick: running `mage operator:generateCrds` with controller-gen 0.4.0 seems to generate some diffs in the generated yaml. better to double check

Reply:
thanks! checked, works as expected. I will keep eye on this.